app/assets/javascripts/admin/addon/controllers/admin-user-index.js

```diff
@@ -369,38 +369,38 @@ export default Controller.extend(CanCheckEmails, {
                 document.location = getURL("/admin/users/list/active");
               }
             } else {
-              bootbox.alert(I18n.t("admin.user.delete_failed"));
+              this.dialog.alert(I18n.t("admin.user.delete_failed"));
             }
           })
           .catch(() => {
-            bootbox.alert(I18n.t("admin.user.delete_failed"));
+            this.dialog.alert(I18n.t("admin.user.delete_failed"));
           });
       };
 
-      const buttons = [
-        {
-          label: I18n.t("composer.cancel"),
-          class: "btn",
-          link: true,
-        },
-        {
-          icon: iconHTML("exclamation-triangle"),
-          label: I18n.t("admin.user.delete_and_block"),
-          class: "btn btn-danger",
-          callback: () => {
-            performDestroy(true);
+      this.dialog.alert({
+        message: I18n.t("admin.user.delete_confirm"),
+        classes: "delete-user-modal",
```

Comment:
That's a nitpick, but I think this should ne named `class`, in html we  do `<div class="foo bar">`